template.md

```diff
@@ -0,0 +1,321 @@
+---
+# required metadata
+
+title: [ARTICLE TITLE | SERVICE NAME]
+description:
+keywords:
+author: [GITHUB USERNAME]
+manager: wpickett
+ms.date: [CREATION/UPDATE DATE]
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.assetid: [GET ONE FROM guidgenerator.com]
+
+# optional metadata
+
+#ROBOTS:
+#audience:
+#ms.devlang: dotnet
+#ms.reviewer: [ALIAS]
+
+---
+
+# Metadata and Markdown Template
+
+This core-docs template contains examples of Markdown syntax, as well as guidance on setting the metadata. To get the most of it, you must view both the [raw Markdown](https://raw.githubusercontent.com/dotnet/core-docs/master/template.md) and the [rendered view](https://github.com/dotnet/core-docs/blob/master/template.md) (for instance, the raw Markdown shows the metadata block, while the rendered view does not).
+
+When creating a Markdown file, you should copy this template to a new file, fill out the metadata as specified below, set the H1 heading above to the title of the article, and delete the content. 
+
+
+## Metadata 
+
+The full metadata block is above (in the [raw Markdown](https://raw.githubusercontent.com/dotnet/core-docs/master/template.md)), divided into required fields and optional fields. Some key notes:
+
+- You **must** have a space between the colon (:) and the value for a metadata element.
+- If an optional metadata element does not have a value, comment out the element with a # or remove it (do not leave it blank or use "na"); if you are adding a value to an element that was commented out, be sure to remove the #.
+- Colons in a value (for example, a title) break the metadata parser. In this case, surround the title with double quotes (for example, `title: "Writing .NET Core console apps: An advanced step-by-step guide"`).
+- **title**: This title will appear in search engine results. You can also add a pipe (|) followed by the product name (for example, `title: Developing Libraries with Cross Platform Tools | .NET Core`). The title doesn't need be identical to the title in your H1 heading and it should contain 65 characters or less (including | PRODUCT NAME).
```

Comment:
There was an issue with the OPS build, but that's fixed now. Ignore my previous comment.